File: app/src/main/java/com/adisa/diningplus/network/entities/Manager.java
package com.adisa.diningplus.network.entities;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

import java.io.Serializable;

@Entity(tableName = "managers")
public class Manager implements Serializable {
    @PrimaryKey
    public int id;
    public String name;
    public String email;
    public String position;
}

File: app/src/main/java/com/adisa/diningplus/network/entities/Manager.kt
package com.adisa.diningplus.network.entities

import org.json.JSONObject

class Manager(raw: JSONObject) {
    var id: Int = raw.getInt("id")
    var name: String = raw.getString("name")
    var email: String? = raw.optString("email")
    var position: String? = raw.optString("position")
}